core/controllers/contributor_dashboard.py

```diff
@@ -307,14 +307,18 @@ def _get_reviewable_exploration_opportunity_summaries(
             for topic in topics
             for reference in topic.get_all_story_references()
             if reference.story_is_published])
-        in_review_suggestions = (
-            suggestion_services.get_reviewable_translation_suggestions(user_id))
-        in_review_suggestion_target_ids = [
+        in_review_suggestions, _ = (
+            suggestion_services
+            .get_reviewable_translation_suggestions_by_offset(
+                user_id, None, None, 0))
```

Comment:
If the payload is too large, would we run into errors?

Reply:
Asking since we have run into issues with a different handler where the payload gets too large that it leads to a request timeout error.

Reply:
Would it be an issue if we're only passing the payload in the backend? The large potential payload is never returned to the frontend (only the limited set of lessons).

Reply:
Probably -- the query() could time out.